### DISCUSSION TITLE
das aktuelle Design der Geräteeinstellungen... sollte verbessert werden

### DISCUSSION THREAD

**Discussion by Baxxy13** [OpenCCU/OpenCCU#1636] [🇩🇪 Generelle Diskussionen]
Da es ja ruhig geworden ist um die Designanpassungen mal die Frage...
Bleibt das jetzt so?

Hat sich mal jemand die Geräteeinstellungen angeschaut, speziell von Geräten (IP) mit vielen Kanälen?
Da wird so viel Platz vergeudet durch die riesen Abstände zwischen den einzelnen Optionen.
Selbst wenn man den Zoomfaktor des Browsers verringert muss man immer noch viel zu viel scrollen.

Firefox 95.0.2 (64-Bit) knapp Fullscreen 1080p
![GIF 31 12 2021 15-36-48](https://user-images.githubusercontent.com/54898249/147828645-43463e35-0375-46cc-9621-ebd6ada92f9e.gif)

Wenn ich in dem Beispiel noch paar Schalzeitpunkte im Wochenprogramm aktiviere komme ich aus dem Scrollen gar nicht mehr raus.

**Comment by jens-maus** [comment]
> Da es ja ruhig geworden ist um die Designanpassungen mal die Frage... Bleibt das jetzt so?

Dieses Jahr sicherlich ;)
Bzw. Das wird sicherlich in nächster Zeit wieder aufgenommen werden.

> Hat sich mal jemand die Geräteeinstellungen angeschaut, speziell von Geräten (IP) mit vielen Kanälen? Da wird so viel Platz vergeudet durch die riesen Abstände zwischen den einzelnen Optionen.

Das ist mir in der Tat koch nicht aufgefallen das die Abstände da so groß sind. Welcher Browser ist das denn? Müsste ich mir mal genauer anschauen.

**Comment by jp112sdl** [comment]
Hmm, bei mir ist das nicht (auch FF 95.0.2)

Weder bei den BidCos Geräten noch bei meinem one-and-only HmIP Gerät "HmIP-PCBS"
<img width="933" alt="Bildschirmfoto 2022-01-01 um 09 41 26" src="https://user-images.githubusercontent.com/18190139/147847150-d57f4c26-7c0a-495e-9acd-c105593707dd.png">

Eine leere HmIP-Heizungsgruppe sieht auch normal aus.

**Comment by Baxxy13** [comment]
Guten morgen zusammen und ein gesundes Neues...
Bei mir sieht das auf 3 Testsystemen sowie mit allen möglichen Browsern so aus. 
(Firefox 95.0.2 (64-Bit); Chromium Based: Edge, Vivaldi, Brave jeweils aktuellste Version)
Eine leere HM-Heizgruppe auf einem komplett frischen System (Pi3B+ mit Nightly 3.61.7.20211230) sieht so aus:
Edge (Version 96.0.1054.62 (Offizielles Build) (64-Bit)) nicht ganz Vollbild 1080p
![GIF 01 01 2022 11-03-18](https://user-images.githubusercontent.com/54898249/147848278-d6bbb95f-6d25-4a25-86a6-504cce1a9665.gif)
Das sieht alles irgendwie merkwürdig und falsch aus, könnte aber daran liegen das die "leer" ist.

**Comment by Baxxy13** [comment]
Hier noch eine leere IP-Heizguppe auf dem frischen System mit Brave-Browser (Version 1.33.106 Chromium: 96.0.4664.110 (Offizieller Build) (64-Bit))
![GIF 01 01 2022 11-20-14](https://user-images.githubusercontent.com/54898249/147848498-13d43f88-00ed-4e55-b644-4e3a2a1b61f9.gif)

**Comment by Baxxy13** [comment]
Um lokale Browserprobleme auszuschließen hier noch die leere IP-Heizgruppe in der "Handy-Ansicht".
![tmp-1641033071272_copy](https://user-images.githubusercontent.com/54898249/147848769-25ee3406-a360-4c26-9cb5-6ae37fa6f77f.jpg)
![tmp-1641033071240_copy](https://user-images.githubusercontent.com/54898249/147848772-42a24df7-25a8-40c2-a6aa-d494efa1eb65.jpg)

**Reply by jens-maus** [reply]
Wieso kann man denn auf deinen Screenshots hier die einzelnen Tabellenzellen sehen? Das sollte so definitiv nicht zu sehen sein und deutet stark daraufhin das du ggf. irgendeinen Patch oder ähnliches installiert hast der ggf. die CSS verändert hat oder so. Wie sicher bist du das du da keine lokale Änderungen/Anpassungen hast die das ggf. erklären könnten? Bitte mal auf einem komplett frischen System probieren und berichten.

**Reply by Baxxy13** [reply]
> Bitte mal auf einem komplett frischen System probieren und berichten.

Na das ist doch frisch und jungfräulich, da wurde nix angefasst.

> auf einem komplett frischen System (Pi3B+ mit Nightly 3.61.7.20211230)

**Comment by jp112sdl** [comment]
Ach mit dem **Nightly**!
Ja da ist es bei mir auch.

In der Stable 3.61.7.20211218 war es noch in Ordnung.

**Reply by jp112sdl** [reply]
Auch bei BidCos... :(

<img width="971" alt="Bildschirmfoto 2022-01-01 um 12 27 47" src="https://user-images.githubusercontent.com/18190139/147849503-76e82a3e-b5b2-4c89-84b4-eb72c367e614.png">

Muss/Kann ja dann nur mit 4214e9d oder 92a20b0 zusammenhängen

**Reply by jens-maus** [reply]
Ah, das ergibt natürlich Sinn. Na dann kann man das natürlich entsprechend melden und dann schau ich mir das die Tage mal an, klar 😜

**Reply by jens-maus** [reply]
Ok, mit der nächsten nightly build sollte das etwas zu weit aufgezogene Layout der Geräteeinstellungen wieder wie vorher aussehen.

**Reply by Baxxy13** [reply]
Alles klar, dann schaue ich mir das morgen wieder an.

**Reply by Baxxy13** [reply]
Kurze Rückmeldung.
Mit dem heutigen Nightly sieht der Bereich wieder aufgeräumt und kompakt aus. Sehr gut!
